hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -464,7 +464,8 @@
           '''
         },
       ]
-      tags: [// when it's set, it can't be modified until the next advance operation completes
+      tags: [// When it's set, it can't be modified until the next advance operation completes
+             // Also, this CSR can't be programmed until we advance to StInit
```

Comment:
@tjaychen 
In the meeting, we thought we could remove this exclusion, but looks like this CSR is gated until we move to StInit. So, we still need to keep it. Does the comment sound right?

```
keymgr_cfg_en #(
.NonInitClr(1'b0) // clear has no effect until init
) u_sw_binding_regwen (
.clk_i,
.rst_ni,
.init_i(init),
.en_i(lc_keymgr_en[KeyMgrEnSwBindingEn] == lc_ctrl_pkg::On),
.set_i(sw_binding_set),
.clr_i(sw_binding_clr),
.out_o(sw_binding_regwen)
);
```

Reply:
i feel like there's a bug in the design here. Let me have a look. 
The software binding actually needs to locked prior to even initialization. However, the unlock cannot happen until we get beyond the init stage.  Something is not right here.

Reply:
see #8316. Let me fix this quickly.

Reply:
The sw_binding isn't needed for advancing to StInit, so before StInit, it doesn't matter whether we lock it or not. And after advance is done, it will be unlocked anyway. Hence, I feel we don't need to lock sw_binding before initialization.

Reply:
so this is specific to boot rom.  Boot rom when doing initial verification, will lock some values into sw_binding, but will not actually ever init or advance the key manager (partially because "good" entropy will not have been setup at that point). 

So the sw_binding values need to stay locked until waaaay later when keymgr is actually actuated. 

Have a look at the code [here](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/mask_rom/mask_rom.c#L146)

Reply:
does it mean we should not automatically unlock sw_binding after advancing from Reset to Init?

Reply:
yes that is right. Looking at the design right now, that's how it behaves.
So i need to make this point clearer in the docs.
Reset -> Init (does not unlock)
Init -> creatorRoot -> ownerintermediate -> owner

these should all cause unlock.  During disable / invalid we technically should not, but i need to double check if that behavior is correct.

Reply:
I see. Thanks for the explanation.
Once you update the design, I will remove this exclusion.

Reply:
Now this exclusion is deleted as design update is in.